bin/setup

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```

Comment:
So we can merge scripts/first_install into setup, but in another PR, better keep this PR only with rails 4.2 update :-)